nservicebus/endpoints/specify-endpoint-name.md

```diff
@@ -57,19 +57,5 @@ By default the endpoint's input queue name is the same as endpoint's name. The i
 
 snippet:InputQueueName
 
-Additionally, the actual queue name has to be passed to the endpoints that want to communicate with it:
+Note: Changing the input queue may also require adjusting the routing configuration for senders to send messages to the renamed queue. See the [routing documentation](/nservicebus/messaging/routing.md) for further details about routing configuration.
 
-
-### Override in code
-
-snippet:InputQueueOverrideRouting
-
-
-### Override in app.config
-
-snippet:InputQueueOverrideRoutingXml
-
-
-In Versions 5 and below the overridden input queue is passed directly to the routing configuration. In Versions 6 and above the logical routing works on an endpoint name level. The endpoint name is passed to the routing configuration, and a separate API call is required to specify the name of its input queue.
```

Comment:
@SzymonPobiega you can also change the value in the normal routing API, as long as you treat the overriden queue name as the new logical name, so there shouldn't be a difference to v5? Also if you look at line 60, do you think that needs a different text? (but makes me think whether we should include some docs about `SendOptions.SetDestination` on the routing page instead?